### PR TITLE
Fix #2642 - allow Android as supported by default for WebExtensions.

### LINKED ISSUE #2642: After xpi file is uploaded Android checkbox is not active
Steps to reproduce:

1.In AMO home page hover over “Tools”
2.Click on “Submit a New Add-on”
3.After submission page is loaded click on “Select a file...”
4.Upload a webextension for Android platform.

Expected results:
Android checkbox is active.

Actual results:
Android checkbox is not active.

Note/Issues:
Verified on FF49.0a1 (2016-05-11) (Win 7). This issue is reproducing in AMO-stage.
Screenshot to this issue: 
animation

### PR DESCRIPTION
**Pull request by EnTeQuAk**

### CODE REVIEW

**Comment by diox** [comment]
Should the minimum working version for web extensions be the same for Android and Desktop ?

**Comment by EnTeQuAk** [comment]
I talked to andy about that, I'll change it up so that Android starts min-level at 48 instead of 42 as Desktop does.

**Comment by EnTeQuAk** [comment]
Fixed the default version for Android to be 48.0, mind a re-review?

File: src/olympia/devhub/tests/test_views.py [hunk 1]
```diff
@@ -2261,6 +2265,26 @@ def test_desktop_excludes_mobile(self):
         self.check_excluded_platforms('desktop.xpi', [
             str(p) for p in amo.MOBILE_PLATFORMS])
 
+    def test_webextension_supports_all_platforms(self):
+        self.create_appversion('firefox', '*')
+        self.create_appversion('firefox', '42.0')
+        # Android is only supported 48+
+        self.create_appversion('android', '48.*')
+
+        self.check_excluded_platforms('valid_webextension.xpi', [])
```
Comment (diox) [inline src/olympia/devhub/tests/test_views.py]: This test is failing, it's returning `[ANDROID.id]`. Seems bad to me, but I don't see why yet.

Comment (EnTeQuAk) [inline src/olympia/devhub/tests/test_views.py]: opps, fixed it. Somehow I forgot to commit some changes :laughing:

**Comment by diox** [comment]
r+

**Pull request merged**